src/main/kotlin/mate/academy/figure/Figure.kt

```diff
@@ -0,0 +1,80 @@
+package mate.academy.figure
+
+import mate.academy.model.Color
+import mate.academy.service.AreaOfFigureSupplier
+import mate.academy.service.DrawSupplier
+
+sealed class Figure(val color: Color): DrawSupplier {
+}
+
+class Circle(color: Color,
+             private val radius: Double): Figure(color),
+    AreaOfFigureSupplier,
+    DrawSupplier {
+    override fun getAreaOfFigure(): Double {
+        return Math.PI * radius * radius
+    }
+
+    override fun drawFigure() {
+        println("Figure: circle, area: ${getAreaOfFigure()} sq. units," +

```

Comment:
You may use """ triple quote marks to avoid concatenation.
Also please check why concatenation is not so good in case if you not familiar with this topic